src/lib.rs

```diff
@@ -27,75 +27,96 @@
 //! | Haiku            | `/dev/random` (identical to `/dev/urandom`)
 //! | SGX              | [RDRAND][18]
 //! | VxWorks          | `randABytes` after checking entropy pool initialization with `randSecure`
-//! | Web browsers     | [`Crypto.getRandomValues`][14] (see [Support for WebAssembly and asm.js][16])
-//! | Node.js          | [`crypto.randomBytes`][15] (see [Support for WebAssembly and asm.js][16])
+//! | Emcripten        | `/dev/random` (identical to `/dev/urandom`)
 //! | WASI             | [`__wasi_random_get`][17]
 //!
-//! Getrandom doesn't have a blanket implementation for all Unix-like operating
-//! systems that reads from `/dev/urandom`. This ensures all supported operating
-//! systems are using the recommended interface and respect maximum buffer
-//! sizes.
+//! This crate doesn't have a blanket implementation for all `unix` targets that
+//! reads from `/dev/urandom`. This ensures all supported targets are using the
+//! recommended interface and respect maximum buffer sizes.
 //!
 //! ## Unsupported targets
 //!
-//! By default, compiling `getrandom` for an unsupported target will result in
-//! a compilation error. If you want to build an application which uses `getrandom`
-//! for such target, you can either:
-//! - Use [`[replace]`][replace] or [`[patch]`][patch] section in your `Cargo.toml`
-//! to switch to a custom implementation with a support of your target.
-//!
-//! [replace]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-replace-section
-//! [patch]: https://doc.rust-lang.org/cargo/reference/manifest.html#the-patch-section
+//! By default, `getrandom` will not compile on an unsupported target. If you
+//! want to use `getrandom` on such target, you can either:
+//!
+//! - Enable `getrandom`'s `"cpu"` feature
+//!   - This makes `getrandom` fallback to using CPU-specific RNG instructions.
+//!   - Currently only `x86`/`x86_64` targets are suppored. On these targets,
+//!     [RDRAND][18] will be used to generate the random data.
+//!   - If `getrandom` is not a direct dependancy of your crate, you can still
+//!     enable this fallback by adding the following to your crate's
+//!     `[dependancies]` section:
+//!     ```toml
+//!     getrandom = { version = "0.2", features = ["custom"]
+//!     ```
```

Comment:
This isn't really a list, is it? A sub-heading might work better.

Reply:
Fixed